lib/components/src/bar/button.tsx

```diff
@@ -6,10 +6,12 @@ import { auto } from '@popperjs/core';
 interface ButtonProps
   extends DetailedHTMLProps<ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement> {
   href?: void;
+  disabled?: boolean;
```

Comment:
@yannbf Do I need this line or line 14?

Reply:
This one specifically, no. `disabled` is part of `ButtonHTMLAttributes<HTMLButtonElement>` so it should be fine. However what you're trying to achieve with IconButtonProps needs to have both `disabled` and `active` because the component could either be a Button (contains disabled) or a Link (does not contain disabled). We can probably improve the typing part here